pkgs/misc/anarchism/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "anarchism-${version}";
+  version = "15.0";
+  src = fetchurl {
+    url = https://0xacab.org/ju/afaq/repository/master/archive.tar.gz;
+    sha256 = "b081885152628c67c0f6d9101be17095d84684bc9dd2c7d4e50f5744614883ef";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/doc/anarchism/html
+    mkdir $out/share/doc/anarchism/markdown
+    mkdir -p $out/share/applications/
+    cp -r html $out/share/doc/anarchism/
+    cp -r markdown $out/share/doc/anarchism/
```

Comment:
Better just `mv`, that would make it clearer:

```
mkdir -p $out/share/doc/anarchism
mv html markdown $out/share/doc/anarchism
```

Reply:
done.